src/Components/MainTableView/InsertTuple/InsertTuple.tsx

```diff
@@ -75,13 +75,73 @@ class InsertTuple extends React.Component<{token: string, selectedSchemaName:str
    */
```

Comment:
Can you update docks to be more detail what is tupleToInsert is exactly, also the TODO stuff is outdated for this pull request